### ISSUE TITLE
Languages

### ISSUE THREAD

**Issue by stevenbrookslv** [JavidPack/CheatSheet#66]
This may sound really dumb, but is it possible to switch the language of the mod? I see there is different localizations, but the mods text is all in Chinese. The game itself is all English as expected, but for some reason this and the Magic Storage mod are both in Chinese.

**Comment by JavidPack** [comment]
This sounds like you tried to run tmodloader through Proton. You'll need to disable that and delete the dotnet folder in the tmodloader install dir.

**Comment by stevenbrookslv** [comment]
Oh wow okay I am using proton! I suspected that might be it but I wasn’t sure how to go back to running it in the Linux runtime after doing it (setting it just immediately failed to open)

I will delete that directory and try changing it to run from the Linux runtime after that!

Out of curiosity, why does using proton cause this bug? I only switched to proton because the regular version was failing to take input from my controller and someone suggested that would help (and it did, at the expense of me having to figure out where the proton save file folder was and two of my mods inexplicably being in Chinese, lol)
